cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,10 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('/#/login?_k=7d9zy5');
```

Comment:
Cypress do not know what url it should visit. You didn't precise the baseUrl in config.

Reply:
fixed